pkgs/applications/audio/youtube-music/default.nix

```diff
@@ -95,4 +139,4 @@ buildNpmPackage {
     mainProgram = "youtube-music";
     platforms = platforms.all;
```

Comment:
This one is not true

Reply:
fixed

Reply:
So `platforms.linux ++ platforms.darwin ++ platforms.windows;`? I don't think `platforms.all` is that inappropriate, if `pkgsCross.mingwW64.youtube-music` doesn't build, it's clearly a failure of nixpkgs for cross-build support.